plugins/net_plugin/net_plugin.cpp

```diff
@@ -1078,7 +1083,6 @@ namespace eosio {
 
       void cancel_sync_wait();
```

Comment:
Should we decrement `sync_timers_active` in `cancel_sync_wait`? Or do it earlier in `sync_timeout`?

Reply:
I'm not sure I understand.

Reply:
If we cancel the wait, won't we have one fewer timers active so the count `sync_timers_active` will be too high?

Reply:
Oh never mind I see `sync_timeout` always decrements it!